### DISCUSSION TITLE
hii I restart my PC after that my chia GUI not detecting my plots please can anyone help

### DISCUSSION THREAD

**Discussion by AjPanchal02** [Chia-Network/chia-blockchain#6988] [Support]
hii I restart my PC after that my chia GUI not detecting my plots please can anyone help

**Comment by loppefaaret** [comment]
WINDOWS COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```

LINUX COMPLETE CONFIG RESET

Close down the Chia, and open up a terminal, and execute the below commands one at a time:
```
mv ~/.chia/mainnet ~/.chia/old
chia init
mv ~/.chia/old/db ~/.chia/mainnet/
mv ~/.chia/old/wallet ~/.chia/mainnet/
```

**Reply by vgalbur** [reply]
> WINDOWS COMPLETE CONFIG RESET
> 
> Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
> 
> ```
> mv ~\.chia\mainnet ~\.chia\old
> cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
> .\chia.exe init
> mv ~\.chia\old\db ~\.chia\mainnet\
> mv ~\.chia\old\wallet ~\.chia\mainnet\
> ```
> 
> LINUX COMPLETE CONFIG RESET
> 
> Close down the Chia, and open up a terminal, and execute the below commands one at a time:
> 
> ```
> mv ~/.chia/mainnet ~/.chia/old
> chia init
> mv ~/.chia/old/db ~/.chia/mainnet/
> mv ~/.chia/old/wallet ~/.chia/mainnet/
> ```
when i reset the syncronization of the wallet, the address of the xch is changed. Please tell me how can I restore my old XCH address, because now I have 0XCH on new address. Thank You!

**Reply by loppefaaret** [reply]
do you have multiple keys on the machine ? 
the wallet database should stay intact, if you followed each step

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
